browser_history/generic.py

```diff
@@ -191,7 +192,7 @@ class Outputs():
     """
     # All formats added here should be implemented in _format_map
     # Formats added here and in _format_map should be in lowercase
-    formats = ('csv', )
+    formats = ('csv', 'json',)
```

Comment:
`formats` is a tuple. You don't need an extra comma at the end. It is required only if there is exactly on element in the tuple which was the case earlier.

Reply:
sure